src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    private String name;
+    private Colors color;
+    private double area;
+
+    public Figure(Colors color) {
+        this.color = color;
+    }
+
+    public abstract void obtainArea();
```

Comment:
create new interface and move this method there

Reply:
done